modules/local/presto/presto_buildconsensus.nf

```diff
@@ -16,13 +16,18 @@ process PRESTO_BUILDCONSENSUS {
     path("*_R1.log")
     path("*_R2.log")
     path("*.tab"), emit: log_tab
-
+    path "versions.yml" , emit: versions
 
     script:
     def barcode_field = params.cluster_sets ? "CLUSTER" : "BARCODE"
     """
     BuildConsensus.py -s $R1 --bf ${barcode_field} --nproc ${task.cpus} --pf PRIMER --prcons $params.primer_consensus --maxerror $params.buildconsensus_maxerror --maxgap $params.buildconsensus_maxgap --outname ${meta.id}_R1 --log ${meta.id}_R1.log > "${meta.id}_command_log.txt"
     BuildConsensus.py -s $R2 --bf ${barcode_field} --nproc ${task.cpus} --pf PRIMER --prcons $params.primer_consensus --maxerror $params.buildconsensus_maxerror --maxgap $params.buildconsensus_maxgap --outname ${meta.id}_R2 --log ${meta.id}_R2.log >> "${meta.id}_command_log.txt"
     ParseLog.py -l "${meta.id}_R1.log" "${meta.id}_R2.log" -f ID BARCODE SEQCOUNT PRIMER PRCOUNT PRCONS PRFREQ CONSCOUNT
+
+    cat <<-END_VERSIONS > versions.yml
+    "${task.process}":
+        presto: \$( BuildConsensus.py --version | awk -F' '  '{print \$2}' )
```

Comment:
python version could be exported as well for all that pesto processes

Reply:
well the python version is fixed by the presto tool, so that's why I'm not mentioning it here. Otherwise also all other dependencies should also be mentioned, no?

Reply:
Yeah sure, that down to preference I guess.